submissions/vorobeyka/pop-up/css/main.css

```diff
@@ -0,0 +1,188 @@
+* {
+	font: small arial sans-serif;
```

Comment:
Use [Prettier](https://prettier.io/) for CSS too.

Reply:
Okay, thx for feedback.